schema_test.go

```diff
@@ -441,6 +442,30 @@ func TestSchema(t *testing.T) {
 				return nil
 			},
 		},
+		{
+			name: "auth2-with-code-flow.yaml",
```

Comment:
Shouldn't you also teat the password flow?

Maybe I missed something

Reply:
Think you! I can finish this code, but maybe I will write in other repository

Reply:
please let me where I can review it, because I don't get it :grin:

Reply:
@the-pawn-2017 ?

Reply:
> @the-pawn-2017 ?

https://github.com/the-pawn-2017/rest,     Thanks again. I'm only theoretically aware of the GitHub collaboration process, and we have time differences, which can be a waste of your time and patience, and finally, thank you again for your efforts